Commands/deadchat.py

```
@@ -0,0 +1,130 @@
+                continue
+
+            text = msg.get("text", "")
+            if EXCLUDE_TEXT in text:
+                continue
+
+            t = _parse_timestamp(msg.get("timestamp"))
+            if t and start <= t < end:
+                count += 1
+
+        return count
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)